Cython/Compiler/Annotate.py

```diff
@@ -228,7 +232,7 @@ def annotate(match):
             return u"<span class='%s'>%s</span>" % (
                 group_name, match.group(group_name))
 
-        lines = self._htmlify_code(cython_code).splitlines()
+        lines = self._htmlify_code(cython_code, True).splitlines()
```

Comment:
Passing things like `True` and `False` into a function without further context screams for a named keyword argument to make it clear what the value means. However, in this case, it should actually use a language name: `"cython"` vs. `"c"` makes it even clearer what is intended.